### PR TITLE
feat(ui): implementation of mantainance page on the admin section

### LINKED ISSUE #227: Creation of `maintanance` page
### Description

The about `maintenance` page from the `admin` section is currently not available.

#### Steps followed
I want to implement the UI of this page by following these steps.

1. Review the `maintenance` page from the older UI version.
2. Defining the UI design of the targeted page.
3. Implement the UI following the standards of coding style.

#### Expected Result

The `maintenance` page with contents, and  enhanced UI design
compared to the former one.

cc: @shaheemazmalmmd @GMishx

### PR DESCRIPTION
**Pull request by dushimsam**
Signed-off-by: dushimsam <dushsam100@gmail.com>

## Description

Implemented the maintenance page from the Admin section. The page is on the route `/admin/mantainance`

### Changes

1. Created a new directory ,`src/pages/admin/Mantainance/index.jsx`
2. Imported and added the new page in the `src/Routes.jsx` as a part of `AdminLayout`
3. Added a new navigation dropdown Item as a part of the admin dropdowns.
4.  Added the route and initial field values for the new page in `src/constants/routes.js`.

### How to test
1. Go in the project's main directory.
2. run command `npm start`
3. Visit the page at `http://localhost:PORT/admin/mantainance` 
4. You should see something looking like this 

##Screenshots
Selection_006
Selection_007

#### Related Issue:
Fixes #227 

@shaheemazmalmmd @GMishx

### CODE REVIEW

**Comment by GMishx** [comment]
Please consider uploading images directly to PR description. We do not see them due to permission issue
image

File: src/constants/routes.js [hunk 1]
```diff
@@ -1,16 +1,13 @@
...
  SPDX-License-Identifier: GPL-2.0
-
  This program is free software; you can redistribute it and/or
  modify it under the terms of the GNU General Public License
  version 2 as published by the Free Software Foundation.
  This program is distributed in the hope that it will be useful,
  but WITHOUT ANY WARRANTY; without even the implied warranty of
  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
  GNU General Public License for more details.
-
```
Comment (GMishx) [inline src/constants/routes.js:13]: Please revert unnecessary changes.

File: src/pages/Admin/Mantainance/index.jsx [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+    normalizePriority: false,
+    rmvUploads: false,
+    rmvTokens: false,
+    rmvTempTables: false,
+    analyseDb: false,
+    rmvRepoFiles: false,
+    dbReindexing: false,
+    verbose: false,
+    rmvRepoOldFiles1: false,
+    rmvRepoOldFiles2: false,
```
Comment (GMishx) [inline src/pages/Admin/Mantainance/index.jsx:40]: Can these values be exported to `src/constants/constants.js`?

File: src/pages/Admin/Mantainance/index.jsx [hunk 3]
```diff
@@ -0,0 +1,286 @@
...
+
+            <InputContainer
+              type="checkbox"
+              checked={fields.verbose}
+              name="verbose"
+              className="my-3"
+              id="verbose"
+              onChange={(e) => handleChange(e)}
+            >
+              verbose (turns on debugging output).
```
Comment (GMishx) [inline src/pages/Admin/Mantainance/index.jsx:219]:
```suggestion
              Verbose (turns on debugging output).
```

**Review by GMishx — CHANGES_REQUESTED** [review]
Please remove the `.idea` folder from commit.

**Comment by dushimsam** [comment]
I have now resolved all queries.
You can review it.

**Review by GMishx — APPROVED** [review]
Changes looks good. Tested, working as expected.

**Comment by dushimsam** [comment]
that's awesome 👍
thanks

**Pull request merged**